pom.xml

```diff
@@ -198,6 +198,13 @@
                 <artifactId>json-smart</artifactId>
                 <version>2.4.9</version>
             </dependency>
+            <dependency>
```

Comment:
I believe this is not the core prat of this change so can we move it out of this, into new PR? reviewing this PR is quite hard as this is not small change

Reply:
we added this dependency to support adding Jenkins plugins as test dependencies. This allows us to craft pipelines in our integration test.
Do you want me to send a pull request just for the changes in pom.xml?

Reply:
please move to separate PR

Reply:
If the changes to the `pom.xml` are in a separate PR, it will be strange to have additional dependencies in the `pom.xml` without the test that use them.

Reply:
we have sent a pull request for pom.xml changes in #86.